miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleController.java

```diff
@@ -154,14 +154,14 @@ public SampleDto getSample(@PathVariable("id") Long id, UriComponentsBuilder uri
         if (dto.getParentId() != null) {
           // Pass
         } else if (dto.getSampleClassId() == null) {
-          throw new RestException("No parent and no target sample class.");
+          throw new RestException("No parent and no target sample class.", Status.BAD_REQUEST);
         } else {
           SampleClass sampleClass = sampleClassService.get(dto.getSampleClassId());
           if (sampleClass == null) {
-            throw new RestException("Cannot find sample class: " + dto.getSampleClassId());
+            throw new RestException("Cannot find sample class: " + dto.getSampleClassId(), Status.BAD_REQUEST);
           }
           if (!sampleClass.getSampleCategory().equals(SampleAliquot.CATEGORY_NAME)) {
-            throw new RestException("Class and type mismatch.");
+            throw new RestException("Class and type mismatch.", Status.BAD_REQUEST);
```

Comment:
No, but the user shouldn't be able to produce this error. If they do, it's a UI bug. This means that the front end picked a sample class of type aliquot and then wrote the wrong type attribute into the JSON.